### DISCUSSION TITLE
褪黑素真能加重抑郁情绪

### DISCUSSION THREAD

**Discussion by qqqays** [zijie0/HumanSystemOptimization#4] [General]
之前得过抑郁症，吃抗抑郁的药已经恢复，偶尔会复发，持续时间不长。
前段时间失眠，听同事说褪黑素有用，问了医生说吃了没有副作用。
吃了几次，第二天头痛，现在抑郁已经持续挺长时间了

**Comment by qqqays** [comment]
这篇文章给了我很多启发，可能对抗抑郁情绪有很大帮助。
当我抑郁的时候，做任何事情都是负反馈，躺着不动会让我好受一些。
按照文章说法，多巴胺是固定的，肯定是某个地方多巴胺消耗过度，才会造成这种情况。
我现在已经找到了主要问题，算是自己的一个bug，正尝试对抗它。

**Reply by zijie0** [reply]
希望能对你有帮助哈。
我本身没有情绪方面的问题，所以没有深入研究这方面的原理和行动建议。后续如果看到其它相关资料会更新上来。
